QUEST_LV_0200.tsv

```diff
@@ -7739,87 +7739,87 @@ QUEST_LV_0200_20150714_007742	I should've persuaded him slowly, but I think we w
 QUEST_LV_0200_20150714_007743	You are a different person. Even so, no is no.{nl}They are very rude. They know the circumstances well, and Evoniphon...{nl}
 QUEST_LV_0200_20150714_007744	And the father superior is locked in the monastery.{nl}Unless we retrieve the monastery, do not mention about the deportation.
 QUEST_LV_0200_20150714_007745	If retrieving monastery is more important, that's good.{nl}If they approve, we will support with everything we can.{nl}
-QUEST_LV_0200_20150714_007746	You should first help those monks.{nl}They would have more than one trouble. You should show them the favor first.
+QUEST_LV_0200_20150714_007746	You should first help those monks.{nl}They should have more than one person in trouble. You should show them mercy first.
```

Comment:
It doesn't make sense to show the monks mercy, because the player needs to persuade them by doing their favors.